requirements.txt

```diff
@@ -2,3 +2,5 @@ pandas
 httplib2
 google-api-python-client
 oauth2client
+google-cloud
```

Comment:
google-cloud is not the most stable package (I've noticed), I'd require a specific version

Reply:
Fixed and pinned to google-cloud-bigquery to 0.25.0 since 0.26.0 breaks things